ocfweb/announcements/templates/announcements/2017-03-20-hiring.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-sm-8 ocf-content-block">
+            <h3></h3>
+            <p><em>March 20, 2017</em></p>
+            <hr />
+            <p>
+                The OCF is making a big transition to being completely
+                self-staffed next semester, and as part of that process, we are
+                looking to hire students to be OCF Operations staff for the
+                2017-2018 academic year! This includes two paid positions,
+                Operations Assistant and Operations Manager, neither of which
+                require any technical experience. These are very important
+                roles, and without dedicated and passionate students to fill
+                them, the OCF will not be able to stay open next semester.
+            </p>
+
+            <h3>Operations Assistants</h3>
+            <p>
+                Operations Assistants will have the following responsibilities:
+
+                <ul>
+                    <li>Staffing the front desk</li>
+                    <ul>
+                        <li>Maintaining lab security</li>
+                        <li>Providing help to OCF members using the lab</li>
+                        <li>Carrying out regular lab maintenance</li>
+                    </ul>
+                    <li>Working on projects that benefit the OCF</li>
+                    <ul>
+                        <li>Marketing and outreach</li>
+                        <li>Collecting feedback on OCF services</li>
+                        <li>Organizing events taking place in the lab</li>
+                        <li>Researching and applying for additional sources of
+                            funding for the OCF</li>
+                    </ul>
+                    <li>Attending regular meetings with the Operations Manager
+                        and the other Operations Assistants</li>
+                </ul>
+
+                This job position will be paid $13.75 per hour, but will also be
+                less work and more flexible than the Operations Manager
```

Comment:
Well now it says "less responsibilities and be more flexible", so not sure that is much better?